executor/common_usb.h

```diff
@@ -823,6 +823,8 @@ static bool lookup_control_response(const struct vusb_descriptors* descs, const
 
 #if GOOS_linux
 #include "common_usb_linux.h"
+#elif GOOS_netbsd
+// Included in common_bsd.h
```

Comment:
Let's keep the include order consistent for different OSes. Please either include `common_usb.h` into `common_usb_linux.h` too, or move NetBSD USB definitions into a separate file and conditionally include that into `common_usb.h`.

Reply:
Alright, sent a mail to @R3x with a patch to add to this PR.

Reply:
Done